mobile/.expo/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "hostType": "tunnel",
+  "hostType": "lan",
```

Comment:
We don't want to make any changes to the frontend. So can you please change it to its original value.

Reply:
How do we test changes in the admin page & MangoDB?